lib/src/react_client/factory_util.dart

```diff
@@ -102,7 +102,7 @@ dynamic generateChildren(List childrenArgs, {bool shouldAlwaysBeList = false}) {
 
   if (children == null) {
     children = shouldAlwaysBeList ? childrenArgs.map(listifyChildren).toList() : childrenArgs;
-    markChildrenValidated(children);
+    markChildrenValidated(children as List);
```

Comment:
Should this still get cast `as List` if `shouldAlwaysBeList` is false?

Reply:
On the line above, the value chlidren is reassigned to is always a `List`, and markChildrenValidated takes in a `List`, but the static type of children is still `dynamic` so we need a cast here. 

In null safety, with Dart's type inference improvements, `children` actually has a static type of `List`, and this becomes an unnecessary cast warning and ends up getting removed.